weather.js

```diff
@@ -5,17 +5,31 @@ dotenv.config();
 const apiURLPrefix = "http://api.openweathermap.org/";
 
 /**
- * Runs a conditional to distinguish if Zipcode or address string was entered.
+ * Get weather data based on whether Zipcode or address string was entered.
  * @param {string|number}input
  * @returns {Promise<void>}
  */
 async function getWeather(input) {
-    if (isNaN(input)) {
-        const data = await getCoordinates(input);
-        await getWeatherData(`lat=${data[0]["lat"]}&lon=${data[0]["lon"]}`);
-        return;
+    try {
+        formatCheck(input);
+        if (isNaN(input)) {
+            const data = await getCoordinates(input);
+            return printWeather(await getWeatherData(`lat=${data[0]["lat"]}&lon=${data[0]["lon"]}`));
+        }
+        printWeather(await getWeatherData(`zip=${input}`));
```

Comment:
I like all this.
Looks very good to me.
Its succinct without in-lining too much and its clear what each step does.